BunqSdk.Tests/Model/Generated/Endpoint/AvatarTest.cs

```diff
@@ -58,8 +58,8 @@ private static string UploadAvatarAndGetUuid(byte[] fileContentByte)
         {
             var customHeaders = new Dictionary<string, string>
             {
-                {ApiClient.HEADER_ATTACHMENT_DESCRIPTION, ATTACHMENT_DECSRIPTION},
-                {ApiClient.HEADER_CONTENT_TYPE, CONTEN_TYPE},
+                {ApiClient.HeaderAttachmentDescription, ATTACHMENT_DECSRIPTION},
```

Comment:
`ATTACHMENT_DECSRIPTION` is still in caps.

Reply:
I might have missed this ones, changing it.